src/app/[locale]/(pages)/blocco-accesso/page.tsx

```diff
@@ -32,9 +33,11 @@ const ProfileBlock = (): React.ReactElement => {
         <Grid sx={{ maxWidth: '576px' }}>
           {isIDPKnown && <IdpListOnApp />}
           <Typography mb={5}>{t('common.lockaccessinfo')}</Typography>
-          <Button href={ROUTES.PROFILE_BLOCK_SUCCESS} variant="contained" size="medium">
-            {t('profile.lockaccess')}
-          </Button>
+          <Link href={ROUTES.PROFILE_BLOCK_SUCCESS}>
```

Comment:
Here and in the other occurrences we can eliminate this link that wraps the button and directly insert the onclick on the button with the router.push (so, using the userRouter)

Reply:
Done